ext/nnstreamer/tensor_decoder/meson.build

```diff
@@ -114,3 +114,40 @@ static_library('nnstreamer_decoder_image_segment',
   install: true,
   install_dir: nnstreamer_libdir
 )
+
+# protobuf decoder
+if get_option('enable-protobuf')
+  # Compile protobuf schema: nnstreamer.proto
+  pb_comp = find_program('protoc', required: false)
```

Comment:
Make it "required: true" and remove Line 123-125. The behavior will be the same :)

Reply:
Sorry for this late. I updated this to be compatible with meson "feature"